src/App.tsx

```
@@ -24,21 +25,23 @@ function App() {
-          <Route path="/start/:playlistCode" element={<StartGuest />} />
-          <Route path="/create" element={<Create />} />
-          <Route path="/home" element={<Home />} />
-          <Route path="/playlist" element={<Playlist />} />
-          <Route path="/error" element={<Error />} />
-          <Route path="/error/:code" element={<Error />} />
-          <Route path="/offline" element={<Offline />} />
-          <Route path="/test" element={<Test />} />
-          <Route path="*" element={<NotFound />} />
-        </SentryRoutes>
-      </Suspense>
+      <ErrorBoundary key={location.key}>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)